README.md

```diff
@@ -11,6 +11,15 @@ The goal is to be **scalable**, **reliable**, fast and extensible.
 We therefore use a _score-based algorithm_ and _no floating-point arithmetic_ whatsoever.
 
 
+## Features
+
+- Supports billions of participants
+- Supports thousands of proposals
+- Handles default grades (static or normalized)
+- No floating-point arithmetic
+- Room for other deliberators (central, usual)
+
+
 ## Example Usage
 
 Collect the **tallies** for each Proposal (aka. Candidate) by your own means,
```

Comment:
we need to define more properly what is a tally. i dont thnk it is obvious for everyone

Reply:
In french, it's "dépouillement", or "décompte".  You're right, this warrants further explanations.